lib/msf/core/opt_base.rb

```diff
@@ -83,6 +84,12 @@ def required?
       required
     end
 
+    # Returns the conditions.
+    #
+    def conditions?
+      conditions
```

Comment:
With the `attr_accessor :conditions` line below, this doesn't seem necessary and I also don't see where it's used. If that's the case then it should be removed.

Reply:
Done!